developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -55,8 +55,9 @@ Use the following steps to set up PostgreSQL on RPM based distributions:
 
 [source, bash]
 ....
-sudo dnf install postgresql
+sudo dnf install postgresql-server
 sudo postgresql-setup --initdb
+sudo systemctl enable --now postgresql
 sudo -u postgres createuser --createdb $USER
```

Comment:
Thinking out loud:
Since we're considering a move to containers, should we suggest something container-based instead? Something like
```shell
podman run -e POSTGRES_USER=$USER -e POSTGRES_PASSWORD=postgres -e POSTGRES_DB=foreman -p 5432:5432 -d --name foreman-db postgres:16-alpine
sudo dnf install postgresql # For pg_dump
```

Reply:
Today none of this document mentions containers so I'd like to separate those changes.